packages/core/package.json

```
@@ -0,0 +1,45 @@
+{
+  "name": "@blockprotocol/core",
+  "version": "0.0.1",
+  "description": "Implementation of the Block Protocol Core specification for blocks and embedding applications",
+  "keywords": [
+    "blockprotocol",
+    "blocks",
+    "core"
+  ],
+  "homepage": "https://github.com/blockprotocol/blockprotocol#readme",
```

```suggestion
  "homepage": "https://blockprotocol.org",
```

? (unless with stick with `...#readme` everywhere)

---

Fixed in https://github.com/blockprotocol/blockprotocol/pull/323/commits/1660342ca9dd945376f882e19b2f139d7540b958